lib/spoom/cli/bump.rb

```diff
@@ -57,7 +83,35 @@ def bump(directory = ".")
           path
         end.compact.uniq
 
-        Sorbet::Sigils.change_sigil_in_files(files_with_errors, from)
+        undo_changes(files_with_errors, from)
+
+        files_changed = files_to_bump - files_with_errors
+        print_changes(files_changed, from: from, to: to, dry: dry, path: exec_path)
+        undo_changes(files_to_bump, from) if dry
+        exit(files_changed.empty?)
+      end
+
+      no_commands do
+        def print_changes(files, from: "false", to: "true", dry: false, path: File.expand_path("."))
```

Comment:
Is it possible to add a sig to this method and `undo_changes` to get this file back to `typed: strict`? Or does the `no_commands` block cause some hiccup here?

Reply:
It is possible for Sorbet but Thor will complain at runtime about your `no_commands` bein a command and not having a documentation. We need to take a look at this at some point.

One solution is to move those helper method to a real helper like it's done here: https://github.com/Shopify/spoom/blob/master/lib/spoom/cli/helper.rb

You can observe the impact of that limitation on the strictness of spoom's cli modules:

![image](https://user-images.githubusercontent.com/583144/104754619-70a2f180-5727-11eb-975f-fe9c4315f259.png)

(the only one at `typed: strict` is the helper)